components/woocommerce/woocommerce.app.mjs

```diff
@@ -6,6 +6,12 @@ export default {
   type: "app",
   app: "woocommerce",
   propDefinitions: {
+    orderId: {
+      type: "integer",
+      label: "Order ID",
+      description: "ID of the Order",
+      optional: false,
+    },
```

Comment:
Could we use async options for the `orderId` prop? That way it will provide a list of orders that the user can select from.
Also, you don't need to include `optional: false` because props default to being required.
```suggestion
    orderId: {
      type: "integer",
      label: "Order ID",
      description: "ID of the Order",
      async options({ page }) {
        const orders = await this.listOrders(page + 1);
        return orders.map((order) => ({
          label: `Order #${order.id}`,
          value: order.id,
        }));
      },
    },
```